vars/sconsBuild.groovy

```diff
@@ -212,8 +254,10 @@ def call(Map config = [:]) {
         script += "PATH+=:${WORKSPACE}/cov_analysis/bin\n"
         scons_exe = "cov-build --dir cov-int " + scons_exe
     }
+
     script += '''# the config cache is unreliable so always force a reconfig
                  # with "--config=force"
+                 pwd
```

Comment:
Debug needing removed?

But more importantly, shouldn't this all be in a `ci/*` script now?

Reply:
Not sure why we put a comment in an imbedded script.  That does seem odd.
Moving the comment in the next push.

And yes, we want to get rid of building inline scripts.

I am mainly concentrating on adding coverage builds and cleaning up the Jenkinsfile for this PR, so that may need to be a separate PR.

Reply:
My concern wasn't the comments in the inline script but rather the addition of the `pwd` to the script.  Is that debug that should be removed now?

Reply:
I did not realize I was the one that added the pwd.  It will be removed in the next push.